Shared/Tracking/Trackers/shikimori/ShikimoriApi.swift

```diff
@@ -0,0 +1,253 @@
+//
+//  ShikimoriApi.swift
+//  Aidoku
+//
+//  Created by Vova Lapskiy on 02.11.2024.
+//
+
+import Foundation
+
+class ShikimoriApi {
+    private let decoder = JSONDecoder()
+    private let encoder = JSONEncoder()
+    private let useragent = "Aidoku"
+    private let dateformat = "yyyy-MM-dd'T'HH:mm:ss.SSSxxx"
+
+    // Registered under vlapsk1y's Shikimori account
```

Comment:
i think it'd probably be better to have it under my account instead (i can change this at the end before merging)